runtime/optwin.vim

```diff
@@ -1164,6 +1164,8 @@ endif
 call append("$", "iminsert\tin Insert mode: 1: use :lmap; 2: use IM; 0: neither")
 call append("$", "\t(local to window)")
 call <SID>OptionL("imi")
+call append("$", "imstyle\tinput method style, 0: on-the-spot, 1: over-the-spot")
```

Comment:
This option doesn't exist in nvim.

Reply:
Yeah, I was wondering if I should bother. Not sure who actually uses optwin.vim.

I'll change it since you mentioned it.

Reply:
I only mentioned it because of the failed tests. :)